.travis.yml

```diff
@@ -60,33 +60,20 @@ jobs:
       python: '3.7'
       cache: pip
       install:
-        # Initialize services required to run tests
-        - docker-compose up --detach mysql redis rabbitmq
```

Comment:
The command "manage.py check" is failing because there's no database running. Add this back.